api/api.py

```diff
@@ -109,7 +110,7 @@ def _get_test_results_json(
 
         try:
             results = test_run.run_algorithm(iterations, burnin, record_interval)
-        except ValueError:
+        except LociRepeatError:
             return error_response('Loci repeats has an insufficient number of entries', 400)
```

Comment:
Testing this, it doesn't appear to actually be caught/printed in the response; I just get a generic "A problem occurred while the server was processing this data." Also, this method is currently expected to just return the JSON by `post`, not the HTML status code as well (despite what the docstring comment says).

Could you hunt down what's causing that (possibly the error doesn't get caught and a different error is thrown in AlgorithmInstance/etc?), and update the `post` method to use the HTML code from this as well?

Reply:
I fixed the error, but I'm not 100% sure what you mean with the HTML status codes. I removed the only line in _get_test_results_json that returned an html status code (un-necessary, could be handled in post). Is that what you were asking? I don't think there are any HTML status codes from this method that can be returned. Or did you just want me to edit the docstring to reflect this?